docs/api/asyncio_con.rst

```diff
@@ -234,8 +234,51 @@ Connection
             If the results of *query* are desired, :py:meth:`query` or
             :py:meth:`query_one` should be used instead.
 
+    .. py:method:: retry()
+
+        Create a :py:class:`AsyncIORetry` object. This is a way to make
+        reliable transactions. See
+        :ref:`edgedb-python-asyncio-api-transaction` for more info.
```

Comment:
I'd make this description a bit more high level:

```suggestion
        Execute a retryable transaction.
        
        This is the preferred method of running transactions as it
        can auto-retry transactions in case a retryable [LINK] error (like
        a network disconnect) happens.
        
        It is very important to always keep in mind that the code within
        the ``retry()`` block might be executed more than one time.
        
        A ``retry()`` call returns an instance of
        :py:class:`AsyncIORetry`. 
        
        See :ref:`edgedb-python-asyncio-api-transaction` for more info.
```

We need to document which transactions are retryable in the main EdgeDB docs and link them from here.

I'd move the `See :ref:.. for more info` to the very bottom of this section, below the example.

Reply:
cc @elprans 

Elvis take a look at how I phrased it. Any suggestions?

Reply:
I prefer "retriable" instead of "retryable", because we say "transaction is retried", although the Internet seems to be split between the spellings.

As for the description, here's my take:

```suggestion
        Open a retriable transaction loop.
       
        This is the preferred method of initiating and running a database
        transaction in a robust fashion.  The `retry()` transaction loop will
        attempt to re-execute the transaction lop body if a transient error
        occurs, such as a network error or a transaction serialization error.
       
        Returns an instance of :py:class:`AsyncIORetry`.
       
        See :ref:`edgedb-python-asyncio-api-transaction` for more details.
```

Reply:
https://english.stackexchange.com/a/305274

Reply:
That isn't an accepted answer, and I don't agree that "retriable" is necessarily a form of "triable", because, again, it's "retried", not "retryed"

Reply:
Well, the "accepted" one isn't always the right one, since it's not accepted by some "expert", but rather by the one who asked the question in the first place :)  The accepted one doesn't speculate on which spelling is more correct either, so I posted a link to the comment that does.

Reply:
@tailhook Let's merge Elvis' version, replacing retriable with retryable (as our native speakers suggested us to)